api/src/main/java/org/jboss/forge/roaster/model/util/Types.java

```diff
@@ -419,4 +511,30 @@ public static int getArrayDimension(String name)
       return count;
    }
 
+   public static <O extends JavaSource<O>> int getBasicArrayDimension(String name)
```

Comment:
Went in by mistake, I thought I had removed it .. the rebase readded it? :/ easy fix...